src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -299,7 +294,7 @@ public function initialized($id)
         }
 
         if (isset($this->privates[$id])) {
-            @trigger_error(sprintf('Checking for the initialization of the "%s" private service is deprecated since Symfony 3.4 and won\'t be supported anymore in Symfony 4.0.', $id), E_USER_DEPRECATED);
+            return false;
```

Comment:
@nicolas-grekas made a mistake here...  alias ID resolution happens first. We dont do that in `has` etc. 

Although i think we actually should do that =/ (could be a new feature and/or patch 3.3/4; wdyt?)